Sources/ShellOut.swift

```diff
@@ -23,81 +24,114 @@ import Foundation
  */
 @discardableResult public func shellOut(to command: String,
                                         arguments: [String] = [],
-                                        at path: String = ".") throws -> String {
+                                        at path: String = ".",
+                                        printOutput: Bool = false) throws -> String {
     let process = Process()
     let command = "cd \"\(path)\" && \(command) \(arguments.joined(separator: " "))"
-    return try process.launchBash(with: command)
+    return try process.launchBash(with: command, printOutput: printOutput)
```

Comment:
How about injecting a `FileHandle` here instead? That way the API user can decide where to stream to, and you can use that to write tests against this new functionality.